src/django/api/templates/mail/facility_list_rejection_body.html

```diff
@@ -2,32 +2,36 @@
 <html lang="en">
     <head>
         <title>
-            Your contribution to the Open Supply Hub was rejected
+            Your contribution to Open Supply Hub was rejected
         </title>
     </head>
     <body>
         <p>
             Hi there,
         </p>
         <p>
-            We’re writing to let you know that your request to upload a dataset to Open Supply Hub was rejected, due to issues with
-            the data in your list. The details of your list are as follows:
+            We’re writing to let you know that your request to upload a dataset
+            to Open Supply Hub was rejected, due to issues with the data in your
+            list. The details of your list are as follows:
         </p>
         <ul>
             <li>
-                Facility list: {{ facility_list_name }} created on {{ facility_list_created_at }}
+                Facility list: {{ facility_list_name }} created on {{
+                facility_list_created_at }}
```

Comment:
Splitting the template tag across 2 lines caused it to not be replaced properly

```
django_1     |             <li>
django_1     |                 Facility list: azavea-4 created on {{
django_1     |                 facility_list_created_at }}
django_1     |             </li>
```

Reply:
Fixed